### ISSUE TITLE
OneToMany support broken? (or just confusing)

### ISSUE THREAD

**Issue by scriptacus** [Raizlabs/DBFlow#287]
I've setup two classes with what I believe is the same setup from the Queen/Ant example.

If I try to save the "Ant" first, I get an error about "no such table" for the Queen table.  If I try to save the Queen first, I get an error about "no such table" for the Ant table.

I was going to try wrapping this in a transaction, but the Transactions documentation simply refers to an object called "database", which I have no idea how to get.

**Comment by kayvannj** [comment]
You can solve your issue using this post https://github.com/Raizlabs/DBFlow/issues/32#issuecomment-71563557

**Comment by agrosner** [comment]
Fixed in `2.2`

**Comment by snezdoliy** [comment]
Hi, I am now using version `2.2.1` of the lib and getting still some weird behavior. 
If we talk in `Queen --> Ants` example, then: 
1. I am creating `Queen` object from JSON (where it has `ants` as array)
2. I am saving whole `Queen` object to DB using TransactionManager
3. I am getting `Queen` object in DB and its `Ant` objects in DB as well, but `queen_id` key is empty for all the ants. 

I have tried overriding `save` like in #32 so that it was like: 

```
    @Override
    public void save() {
        super.save();
        for (Ant ant : ants) {
            ant.associateQueen(this);
            ant.save();
        }
    }
```

And then I am getting exception like:

```
    java.lang.RuntimeException: com.raizlabs.android.dbflow.structure.container.ForeignKeyContainer$InvalidMethodCallException: Cannot call save() on a foreign key container. Call load() instead

```
